docs/processes/release/tasks/standards.md

```diff
@@ -0,0 +1,72 @@
+<!--
+© 2021 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Standards Updates
+
+## Time Zones
+
+Make sure the timezone (Olson) data is up to date.
+
+Olson timezone data should be regularly updated in the trunk. If the data is out
+of date, follow the instruction described in the
+[readme (BROKEN LINK)](http://source.icu-project.org/repos/icu/data/trunk/tzdata/readme.txt)
+file.
+
+Once the timezone resource file is generated by the process above, the file
+should be checked into
+[zoneinfo64.txt](https://github.com/unicode-org/icu/blob/master/icu4c/source/data/misc/zoneinfo64.txt)
+in the trunk.
+
+## Unicode
+
+### Update Unicode Data
+
+See the [Unicode Update page](../../uni-up.md) and the instructions in
+[icu/trunk/source/data/unidata/changes.txt](https://github.com/unicode-org/icu/blob/master/icu4c/source/data/unidata/changes.txt)
+and nearby files, also check the change log for this and related files such as
+UnicodeData.txt.
+
+### Update script code enums
+
+Update script code enums with new codes from ISO 15924, send a notice for them
+as public API (born @stable) to the icu-design list.
```

Comment:
should the `@` be escaped? it shows up in red.

Reply:
Looks like it's an "@mention". Backslash escaping apparently doesn't work, backtick apparently does: https://github.com/github/markup/issues/1168